sklearn/datasets/species_distributions.py

```diff
@@ -72,9 +72,8 @@ def _load_coverage(F, header_length=6, dtype=np.int16):
     header = dict([make_tuple(line) for line in header])
 
     M = np.loadtxt(F, dtype=dtype)
-    nodata = header[b'NODATA_value']
+    nodata = int(header[b'NODATA_value'])
```

Comment:
What's going on here? Does this fix make a practical difference that should be recorded as a bug fix?